src/main/java/com/iexec/common/web/ApiResponse.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2021 IEXEC BLOCKCHAIN TECH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.iexec.common.web;
+
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+import java.util.List;
+
+@Getter
+@Builder
+@EqualsAndHashCode
+@ToString
+public class ApiResponse<T> {
```

Comment:
```suggestion
public class ApiResponseBody<T> {
```
?
Since it's more of a body than a hole response.

Reply:
Sure, that's more explicit. Thanks!